src/v2/ion/uiSchemaLabelTransformer.js

```diff
@@ -0,0 +1,143 @@
+ */
+
+import { _, loc, $ } from 'okta';
+import Bundles from 'util/Bundles';
+import Logger from 'util/Logger';
+
+const SECURITY_QUESTION_PREFIX = 'enroll-authenticator.security_question.credentials.questionKey.';
+
+const I18N_OVERRIDE_MAPPINGS = {
+  'identify.identifier': 'primaryauth.username.placeholder',
+  'identify.credentials.passcode': 'primaryauth.password.placeholder',
+  'identify.rememberMe': 'remember',
+
+  'select-authenticator-enroll.authenticator.email': 'oie.email',
+  'select-authenticator-enroll.authenticator.password': 'oie.password',
+  'select-authenticator-enroll.authenticator.phone': 'oie.phone',
+  'select-authenticator-enroll.authenticator.security_key': 'oie.webauthn',
+  'select-authenticator-enroll.authenticator.security_question': 'oie.security.question',
+
+  'authenticator-enrollment-data.phone.authenticator.phoneNumber': 'mfa.phoneNumber.placeholder',
+
+  'enroll-authenticator.password.credentials.passcode': 'oie.password.passwordLabel',
+  'enroll-authenticator.security_question.sub_schema_local_credentials.0': 'oie.security.question.questionKey.label',
+  'enroll-authenticator.security_question.sub_schema_local_credentials.1': 'oie.security.question.createQuestion.label',
+  'enroll-authenticator.security_question.credentials.answer': 'mfa.challenge.answer.placeholder',
+  'enroll-authenticator.security_question.credentials.question': 'oie.security.question.createQuestion.label',
+  'enroll-authenticator.security_question.credentials.questionKey': 'oie.security.question.questionKey.label',
+
+  'select-authenticator-authenticate.authenticator.email': 'oie.email',
+  'select-authenticator-authenticate.authenticator.password': 'oie.password',
+  'select-authenticator-authenticate.authenticator.phone': 'oie.phone',
+  'select-authenticator-authenticate.authenticator.security_key': 'oie.webauthn',
+  'select-authenticator-authenticate.authenticator.security_question': 'oie.security.question',
+
+  'challenge-authenticator.email.credentials.passcode': 'mfa.challenge.enterCode.placeholder',
+  'challenge-authenticator.password.credentials.passcode': 'mfa.challenge.password.placeholder',
+  'challenge-authenticator.phone.credentials.passcode': 'mfa.challenge.enterCode.placeholder',
+  'challenge-authenticator.security_question.credentials.answer': 'mfa.challenge.answer.placeholder',
+};
+
+const getI18nKey = (i18nPath) => {
+  let i18nKey;
+  if (i18nPath.indexOf(SECURITY_QUESTION_PREFIX) === 0) {
+    const securityQuestionValue = i18nPath.replace(SECURITY_QUESTION_PREFIX, '');
+    i18nKey = `security.${securityQuestionValue}`;
+  }
+
+  if (I18N_OVERRIDE_MAPPINGS[i18nPath]) {
+    i18nKey = I18N_OVERRIDE_MAPPINGS[i18nPath];
+  }
+
+  if (i18nKey && !Bundles.login[i18nKey]) {
+    Logger.warn(`expect i18n key ${i18nKey} for ${i18nPath} but not found in 'login' bundle.`);
+  }
+
+  return i18nKey;
+};
+
+const overrideLabelIfI18NPathExists = (i18nPath, value) => {
```

Comment:
i would change the name of this function to something general.. because it returns `value` as well when i18nkey not exists that means not overriding ?

Reply:
yeah, Ani made similar comment above and I'll tweak the function implementation accordingly.